.circleci/config.yml

```diff
@@ -69,6 +73,9 @@ workflows:
             branches:
               ignore: 
                 - master
+          extra_build_args: --cache-from=sourcecred/sourcecred:latest
```

Comment:
The current build https://circleci.com/gh/good-labs/sourcecred/50 is not using caching.
I believe this is because you need to include the base image that's in the Dockerfile `FROM`.
So `--cache-from=node:12,sourcecred/sourcecred:latest`.

Reply:
As well as pull this image. In the earlier commands.

Reply:
Gotcha! I wasn't sure if node would be included as a layer, since it's part of sourcecred/sourcecred. I'll add right away.

Reply:
Yeaah I think this is a pretty unintuitive aspect of `--cache-from`. My guess is this has to do with switching from implicit whatever we have locally to explicit cache _only_ from here. Which then fails on the very first layer because we don't have node:12 locally.

Reply:
It must use the entire image id. Anyway, green now! :green_apple: :green_book: :green_heart: